scripts/superdesk-desks/desks.js

```diff
@@ -280,31 +279,26 @@
                 scope.fetchPrevious = function() {
                     if (!scope.fetching && scope.page > 2) {
                         scope.fetching = true;
-                        scope.page = scope.page - 1;
-                        if (scope.page > 2) {
-                            criteria.source.from = (scope.page - 3) * criteria.source.size;
-                        } else {
-                            criteria.source.from = 0;
-                        }
+                        scope.page -= 1;
+                        criteria.source.from = scope.page > 3 ? (scope.page - 3) * criteria.source.size : 0;
```

Comment:
I think it would be very helpful to add an inline comment and describe what happens here, when navigating backwards from page numbers > 3.